<!-- longitudinal_controller -->
<launch>
  <node pkg="track_manager" type="track_manager_node" name="track_manager" output="screen">
    <rosparam file="$(find track_manager)/params/rock.yaml" command="load" />
  </node>
  <!-- load urdf -->
  <group ns="vehicle">
    <param name="robot_description" textfile="$(find track_manager)/urdf/e100_1.urdf"/>
    <node pkg="robot_state_publisher" type="robot_state_publisher" name="vehicle_state_publisher"/>
  </group>
  <group ns="target">
    <param name="robot_description" textfile="$(find track_manager)/urdf/target.urdf"/>
    <node pkg="robot_state_publisher" type="robot_state_publisher" name="target_state_publisher"/>
  </group>
</launch>
